lib/season_stats.rb

```diff
@@ -6,9 +6,12 @@ def initialize(games, teams, game_teams)
     super(games, teams, game_teams)
   end
 
-  def shot_accuracy_by_team_id(team_id)
-    (total_goals_by_team_id(team_id).to_f/total_shots_by_team_id(team_id) * 100.0)
-    .round(2)
+  def most_accurate_team(season_id)
+    find_name(shots_and_goals_per_team(season_id).max_by { |team, ratio| ratio}.first)
```

Comment:
Good word (ratio) to describe what the BV is. I'll implement it in my similar methods, too.